bundles/org.openhab.binding.onewiregpio/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -22,6 +22,13 @@
 				<default>600</default>
 			</parameter>
 
+			<parameter name="precision" type="integer">
```

Comment:
Please add the bounds also here

Reply:
If I add bounds here, do I have to validate that the number is in these bounds in the code?

Reply:
I think it makes sense to keep the bounds there, I'm in doubt whether it was used only for the UI or also enforced for textual configuration.

Reply:
Hm... I just looked into the xsd and I'm not quite sure how to do that. Is it through options and writing down the possible values as string with limitToOptions=true afterwards or there is easier way to say it's integer with bounds 0-3?

Reply:
Sorry I was a little bit sick the last few days so I failed to answer :-)

https://github.com/openhab/openhab-addons/blob/88c489091862784167ce0880c4fbde2d8a0b6c0c/bundles/org.openhab.binding.sensebox/src/main/resources/ESH-INF/config/config-description.xml#L15

Just provide `min` and `max`